app/views/application/personalDetails.scala.html

```diff
@@ -133,44 +134,33 @@ <h3 class="heading-medium">Qualification requirements</h3>
                   <tr>
                     <td>
                       <ul class="">
-                        @availableSchemes.filter(scheme => !scheme.requiresALevel && !scheme.requiresALevelInStem).map { scheme =>
-                          <li>@scheme.name</li>
-                        }
+                        <li>Business</li>
```

Comment:
@charge-valtech Does this replace automatic scheme listing with hard coded data?